portugues/ladrao/assalto.md

```diff
@@ -0,0 +1,7 @@
+O ladrão chega com a arma apontando no seu peito e fala "vou te roubei". O que você faz?
+
+[Dou uma esquivada estilo Bruce Lee e retruco "vou te peguei"](ladrao/res/bruce.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.